pkg/apis/certificates/fuzzer/fuzzer.go

```diff
@@ -32,7 +32,7 @@ var Funcs = func(codecs runtimeserializer.CodecFactory) []interface{} {
 	return []interface{}{
 		func(obj *certificates.CertificateSigningRequestSpec, c fuzz.Continue) {
 			c.FuzzNoCustom(obj) // fuzz self without calling this function again
-			obj.Usages = []certificates.KeyUsage{certificates.UsageKeyEncipherment}
+			obj.Usages = []certificates.KeyUsage{certificates.UsageDigitalSignature}
```

Comment:
revert this line, it's unrelated to the kubelet usage changes

Reply:
updated.